Content.Server/Atmos/Piping/Unary/Components/GasVentPumpComponent.cs

```diff
@@ -135,6 +135,10 @@ public float InternalPressureBound
         [ViewVariables(VVAccess.ReadWrite)]
         [DataField("depressurizePressure")]
         public float DepressurizePressure = 0;
+
+        [ViewVariables(VVAccess.ReadWrite)]
```

Comment:
replace these 2 lines with just [DataField]
also false is the default bool so you dont have to specify it

Reply:
Done. Also didn't realize that about bools, good to know. I do want to leave it defined though, just for clarity.